src/utils/webrtc/MediaDevicesManager.js

```diff
@@ -200,14 +212,14 @@ MediaDevicesManager.prototype = {
 					// Couldn't find device by id
 					console.debug('Could not find previous audio device, falling back to default/first device in the list', BrowserStorage.getItem('audioInputId'), this.attributes.devices)
 				}
-				this.attributes.audioInputId = this._fallbackAudioInputId
+				this.attributes.audioInputId = getFirstAvailableMediaDevice(devices, this._preferenceAudioInputList, this._fallbackAudioInputId) ?? 'default'
```

Comment:
`this._fallbackAudioInputId` can be omitted given that we have the list now, but let's do it next time.